ansible/roles/load_test/templates/pgbench-test.j2

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+#
+# {{ ansible_managed }}
+#
+
+scales=( 1 10 100 1000 3000 5000 10000 )
+start=$(date)
+for i in `seq {{ iterations }}`
+do
+    for j in `seq {{ reruns }}`
+    do
+        out=""
+        conn_options=" -U postgres -h {{ groups['load_test_host'][0] }} -p 5432 "
+
+        # Initialize database
+        scale=$((scales[$i - 1]))
+        pgbench{{ postgresql_version }} $conn_options -i -s $scale {{ db_name }}
+
+        # Run actual test
+        {% if item == "read" %}
+        out+=$(pgbench{{ postgresql_version }} $conn_options -c {{ connections }} -j {{ threads }} -T {{ time_in_sec }} -S {{ db_name }})
+        {% elif item == "writeread" %}
+        out+=$(pgbench{{ postgresql_version }} $conn_options -c {{ connections }} -j {{ threads }} -T {{ time_in_sec }} {{ db_name }})
```

Comment:
@snopoke when this is deployed, the wiki [tests](https://wiki.postgresql.org/wiki/Pgbenchtesting) are created under the `writeread` script. (should already be deployed but the receiver email is wrong and you may want to change the T for the first few trails). The reason I went with this approach is so we see a graph of all the different scales (1, 10, 100, 1000, 3000, 5000, 10000) rather than using the formula provided in pgwiki. i did this cause i wasn't sure the wiki formula would be correct given that it might depend on more parameters than the wiki suggests. don't know if that is a real concern though